_includes/sections/working-here/your-pay-pension-and-other-benefits.md

```diff
@@ -108,38 +108,100 @@ your holiday allowance shown on your BreatheHR dashboard.
-recorded in BreatheHR, and update it if necessary. If you're sick for more than
-a week, you'll also need to provide a
-[fit note](https://www.gov.uk/government/publications/the-fit-note-a-guide-for-patients-and-employees).
-
-If you're sick on more than 10 occasions or for more than 10 days over a
-12-month period, we'll invite you to an absence meeting to discuss your
-situation. We may agree with you an action plan and/or a review period. We may
-seek medical advice. Should your attendance fail to improve after a three-month
-period, we will invite you to a further absence meeting to discuss your
-situation. In this meeting we will also give consideration to your possible
-dismissal.
-
-We'll also invite you to an absence meeting if you're off sick for more than
-three weeks on a single occasion. In addition to the above, we will seek to
-support your return to work and discuss with you any temporary or permanent
-adjustments which may assist you. If you are unable to return to work in your
-current role we will consider if there is a suitable alternative role. If we are
-unable to reach an agreement that will permit you to return to work, we will
-give consideration to your possible dismissal.
-
-We will always only consider dismissal after we have exhausted all other
-reasonable options.
+This policy covers both short and longer term sickness absences. Our aim is to
+look after the health and wellbeing of all dxw staff and the needs of the
+company.
+
+##### Communication of sickness
+
+If you’re sick, you must let your line manager or a member of the HR team know
+by 10:00am or as soon as you reasonably can after that. This can be done via
+email, directly on Slack, phone call or text. Line managers will record and
+monitor sickness for individuals within their teams.
+
+If you are unwell and know that you are going to be off for a period of time, or
+have been signed off by your GP until a certain date, you don’t need to update
+your line manager daily. Let them know when you will be checking in again so
+they know when to expect to hear from you.
+
+When you return to work, you must check your sickness has been correctly
+recorded in BreatheHR, update it if necessary and mark yourself as returned to
+work. Your line manager will then review and close the sick leave. If you're
+sick for more than 7 calendar days in a row, you'll also need to provide a fit
+note.
+
+##### Absence meetings
+
+Your line manager should check-in with you on a regular basis about your
+wellbeing, and catch up with you after any sickness absence to see how you are.
+If you're sick for more than 7 working days over a rolling 12 month period, your
+line manager will invite you to an absence meeting to discuss your health. Where
+appropriate, we’ll talk about how we can support your return to work and any
+temporary or permanent adjustments which might help improve your wellbeing and
+reduce your levels of absence. We might also agree an action plan and / or a
```

Comment:
```suggestion
temporary or permanent adjustments which might help improve your wellbeing and
support you to need fewer absences. We might also agree an action plan and / or a
```

The current wording talks like the goal is reduced absence rather than a reduced need for absence. My suggested wording is a bit clumsy, though.